### DISCUSSION TITLE
Exception on Setting Constraint

### DISCUSSION THREAD

**Discussion by narwahls** [netbox-community/netbox#8025] [Q&A] [answered]
We are evaluating Netbox to be used by multiple Teams.
Each Team should only see its own Devices, VMs and so on.

When I want to set the Constraint:
```json
{"tenant__id": 39}
```

I always get the following exception:
![grafik](https://user-images.githubusercontent.com/4248315/145404107-60f6a75e-3fda-4c55-b545-fccbd911bc39.png)

I am using NetBox 3.1 hosted on Kubernetes.
Any hints as to why I am getting this error or hints on turning the log-level up?

**Comment by kkthxbye-code** [comment, marked as answer]
I can't replicate this with the filter you have used. For anyone to realisticly help you, you need to provide exact steps to replicate it. It's especially important to note what components the permission is applied to.

**Reply by candlerb** [reply]
Also, please provide the full backtrace.  You can get this by setting ADMINS and EMAIL in `configuration.py`, and then it will be E-mailed to you when it occurs.

**Comment by narwahls** [comment]
New Year, new try.

As I tried it again, the constraint was set correctly.
This Q&A can be closed.
Many thanks for your Support.
